src/DateTimeDefault.ts

```diff
@@ -48,7 +48,10 @@ export function defaultParseToDateTime(dateTimeString: string, dateFormat: strin
 
 export const secondsExtendedRegexp = /^ss\.(s+|0+)$/
 
-export function defaultParseToTime(timeItems: string[], timeFormat: string): Maybe<SimpleTime> {
+function defaultParseToTime(timeItems: string[], timeFormat: Maybe<string>): Maybe<SimpleTime> {
+  if(timeFormat === undefined) {
+    return undefined
+  }
```

Comment:
Codecov says this condition is not covered by tests. Please add one

Formatting:
```suggestion
  if (timeFormat === undefined) {
    return undefined
  }
```